queue-event-handler.js

```diff
@@ -1,18 +1,40 @@
 const postal = require('postal');
 const Github = require('./github');
 const logger = require('./logger');
-const { tail } = require('ramda');
+const { slice } = require('ramda');
 const pullRequestManager = require('./manager-pullrequest');
 const queueManager = require('./manager-queue');
 const { unpackQueueName } = require('./helpers/queue-helpers');
 
-const addItem = ({ queue }) => {
+const MAX_EXPLICIT_QUEUE_LENGTH = 3;
```

Comment:
I'm not sure about the const name

Reply:
Are you sure about `3`? I'd go for `5` while we gather some data

Reply:
Not sure about the const name either 🙁 will try to improve it, any idea?
`5` is as good as `3` at this point I guess, let's go with it.

Reply:
I don't get the `EXPLICIT` part TBH